docs/source/release_notes/lts/next.rst

```diff
@@ -88,6 +88,118 @@ introduced by the next LTS version are:
   might still modify this to better follow their conventions (eg.
   for RPM the original locations are used)
```

Comment:
Hmm, it's not part of this PR, but it'd be nice to include the hierarchy of those files (to safe time to people that are reading it) as even `/etc` is respected.